requirements/edx/paver.txt

```diff
@@ -5,3 +5,4 @@ lazy==1.1
 path.py==7.2
 watchdog==0.7.1
 python-memcached
+libsass==0.10.0
```

Comment:
This will mean that the system packages will also need to be installed right?  Is there a corresponding update to the configuration repo to add libsass to the list of system packages installed for edx-platform?